sample_projects/test_flag_passing/test.nix

```diff
@@ -0,0 +1,10 @@
+{ pkgs? import ../../nixpkgs.nix { config = {}; }
+, generatedBuild ? ./Cargo.nix { } }:
+
+let instantiatedBuild = pkgs.callPackage generatedBuild {};
+in instantiatedBuild.rootCrate.build.override {
+    runTests = true;
+    testCrateFlags = [
+      "--skip" "this_must_be_skipped"
```

Comment:
Shouldn't this be in one argument as otherwise it would also work before this change?

Reply:
Before this change the test would have failed regardless of this being a single (`--skip this_must_be_skipped` interpreted as the pattern in its entirety and all tests are skipped) or multiple arguments (`--skip` is interpreted as the pattern and no functions match, `this_must_be_skipped` is ignored entirely).

Reply:
```
$/tmp/xxx/target/debug/deps/xxx-04aa82fa44a00818 -- '--skip' 'it_works'

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out

$/tmp/xxx/target/debug/deps/xxx-04aa82fa44a00818 -- '--skip it_works'

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 1 filtered out
```